### ISSUE TITLE
Fix for issue  "Fix GoSec issues (#506)" broke the maven gav reader

### ISSUE THREAD

**Issue by johanglarsson** [jfrog/frogbot#515]
### Describe the bug

The function `runMvnCommand` in maven package handler and its handling of the command output changed in this fix. This lead to the maven-gav-reader:gav producing empty result. Probably due to its new handling of the command output.

When I changed to use the command:

```
if err = mvnutils.RunMvn(mvnParams); err != nil {
    return
}
readerOutput = make([]byte, buf.Len())
_, err = io.ReadFull(&buf, readerOutput)
return
```
Instead of this fix:

```
if err = mvnutils.RunMvn(mvnParams); err != nil {
  readerOutput = make([]byte, 0)
  _, _ = io.ReadFull(&buf, readerOutput)
  if len(readerOutput) > 0 {
	// Log output if exists
	log.Info(string(readerOutput))
  }
  err = fmt.Errorf("failed running command 'mvn %s': %s", strings.Join(goals, " "), err.Error())
}
return
```
The maven-gav-reader started working. I guess that the condition in the current code is wrong. It only reads the buffer if the error stream is populated. 

### Current behavior

17:57:27 [Error] the following errors occured while fixing vulnerabilities in /tmp/jfrog.cli.temp.-1695318942-1310474772:
couldn't find any pom.xml files in the current project

### Reproduction steps

Just do the scan-repository on the current repository which has a maven pom.

We used a simple spring boot project.

### Expected behavior

No error, it should continue.

### JFrog Frogbot version

2.15.0

### Package manager info

Maven

### Git provider

GitLab

### Operating system type and version

Red Hat UBI 8

**Comment by sverdlov93** [comment]
Hi @johanglarsson 
Thanks for reporting this issue.
Our apologies for the inconvenience caused by this.
This [PR](https://github.com/jfrog/frogbot/pull/517) should fix it.
I will update this thread once the next release will be out.

**Comment by EyalDelarea** [comment]
Hey @johanglarsson,
New version of Frogbot has been released, `v2.16.0` which includes a fix for the reported issue.
Please inform us if your problem has been resolved.
Thank you for your cooperation!

**Comment by johanglarsson** [comment]
It seems like the maven-gav-reader has disappeared from releases.jfrog.io.
